packages/jest-types/src/Global.ts

```diff
@@ -48,7 +48,7 @@ export interface Describe extends DescribeBase {
   skip: ItBase;
 }
 
-export interface Global {
+export interface Global extends NodeJS.Global {
```

Comment:
I didn't know this was the fix. I tried extending the global inside the `declare`. Cool beans.